securedrop/sass/_base.sass

```diff
@@ -71,10 +71,6 @@
 @import "modules/grid-item"
 // Option - might be unused. Delete?
 @import "modules/option"
-// Flexbox - support for flexbox layouts
-@import "modules/flexbox"
```

Comment:
The flexbox module is used elsewhere, so needs to be kept. Not sure how it got swept up in the revert, as it was added in https://github.com/freedomofpress/securedrop/commit/4ae0c3f3fded39f5eb6837bb7ccc74a29bd003e3#diff-e95835e98013102be1f7b1c88d5505e0.

Reply:
If I remember correctly, the revert triggered a conflict; I must have accidentally removed it on resolution. Will amend.